packages/core/src/js/feedback/integration.ts

```diff
@@ -0,0 +1,31 @@
+import type { Integration } from '@sentry/core';
+
+import { defaultConfiguration } from './defaults';
+import defaultStyles from './FeedbackForm.styles';
+import type { FeedbackFormProps } from './FeedbackForm.types';
+
+export const FEEDBACK_FORM_INTEGRATION_NAME = 'Feedback Form';
```

Comment:
To match `MobileReplay`.

````suggestion
export const FEEDBACK_FORM_INTEGRATION_NAME = 'MobileFeedback';
````